src/Microsoft.DocAsCode.Build.Engine/MarkdownProviders/JsonTokenTreeServiceProvider.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Composition;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+
+    [Export("jsonTokenTree", typeof(IMarkdownServiceProvider))]
```

Comment:
This is the GfmServiceProvider
[Export("gfm", typeof(IMarkdownServiceProvider))]

---
In reply to: [74901358](https://github.com/dotnet/docfx/pull/608#discussion_r74901358) [](ancestors = 74901358)